antiplag/tests/tests_email.py

```diff
@@ -0,0 +1,47 @@
+from django.test import TestCase
+from django.conf import settings
+from django.test import Client
+
+# Create your tests here.
+
+
+class EmailTestCase(TestCase):
+    def test_invalid_email(self):
+        """Verify that an invalid email returns correct error message"""
+        c = Client()
+        response = c.post(
+            "/api/submissions/",
+            {"email": "invalid", "text": "This is my text"},
+            content_type="application/json",
+        )
+        self.assertEqual(
+            response.content.decode("utf-8"),
+            '{"error":"Unrecognized email address format."}',
+        )
+        self.assertEqual(response.status_code, 400)
+
+    def test_valid_email(self):
+        """Verify that a valid email passes through validation"""
+        c = Client()
+
+        response = c.post(
+            "/api/submissions/",
+            {"email": "test@gmail.com", "text": "This is my text"},
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, 201)
+
+    def test_no_email(self):
+        """Verify that a no email passes through validation"""
+        c = Client()
+
+        response = c.post(
+            "/api/submissions/",
+            {"text": "This is my text"},
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, 201)
+
+    def test_email_API_key(self):
```

Comment:
Please handle the test case - I'm not sure, but I think if `EMAIL_HOST_PASSWORD=` is present in `.env`, empty value would be represented as empty string, therefore this test would test all the time.